omod/src/main/resources/apps/adminui_configuremetadata_admin_links_extension.json

```diff
@@ -47,6 +47,18 @@
         }
     },
 
+    {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.visits",
+        "extensionPointId": "${project.parent.groupId}.${project.parent.artifactId}.adminLinks",
+        "type": "link",
+        "label": "adminui.manageVisitTypes.title",
+        "url": "adminui/metadata/visits/visittypes/manageVisitTypes.page",
+        "requiredPrivilege": "Manage Visit Types",
+        "extensionParams":{
+            "group": "org.openmrs.module.adminui.visits.group"
```

Comment:
Use a maven placeholders for the package and moduleId

Reply:
Yes, I will update with maven placeholders